tests/jballerina-unit-test/src/test/resources/test-src/types/constant/constant-type-negative.bal

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+const int CI6 = 1 + 2;
+const float CF1 = 1.0 + 2.0;
+const decimal CD1 = 1.0 + 2.0;
+// const byte CBT1 = 1; // Uncomment after fixing #33889
```

Comment:
For when the expression is a literal we can still add tests, right? Without waiting for `33889` to be fixed?